gnovm/tests/files/zrealm_crossrealm22a.gno

```diff
@@ -0,0 +1,27 @@
+// PKGPATH: gno.land/r/crossrealm_test
+package crossrealm_test
+
+import (
+	"std"
+
+	"gno.land/r/demo/tests/crossrealm"
+	"gno.land/r/demo/tests/crossrealm_b"
+)
+
+// TODO: consider this and xxx22.gno, xxx21.gno, xxx20.gno
+// how about func()int? and func() crossrealm.Fooer?
+// check type for param type?
+// see also 37a.gno
+
+func main() {
+	f := crossrealm_b.Fooer
+	crossrealm.SetFooerGetter(func() crossrealm.Fooer {
```

Comment:
the `funcLit` is not attached to current realm, nor associated to any real object owned by current realm, so it's potentially valid to be attached to external realm, if does not contain any object not belongs to the target external  realm.